src/core/chat-api.ts

```diff
@@ -85,6 +92,12 @@ interface DeleteMessageParams {
   metadata?: MessageOperationMetadata;
 }
 
+export interface AddMessageReactionParams {
```

Comment:
Appreciate its internal, but might be worth a comment on the type?

Reply:
done